lib/auth/auth_with_roles.go

```diff
@@ -3610,7 +3610,7 @@ func (a *ServerWithRoles) GenerateDatabaseCert(ctx context.Context, req *proto.D
 		if err := a.canImpersonateBuiltinRole(types.RoleDatabase); err != nil {
 			log.WithError(err).Warnf("User %v tried to generate database certificate but is not allowed to impersonate %q system role.",
 				a.context.User.GetName(), types.RoleDatabase)
-			return nil, trace.AccessDenied("access denied")
+			return nil, trace.AccessDenied("access denied. This user is not authorized to create certificates, follow the following guide https://goteleport.com/docs/database-access/reference/cli/#tctl-auth-sign.")
```

Comment:
We know the exact problem here, would be nice to just share it imo
```suggestion
			return nil, trace.AccessDenied(`access denied. The user must be able to impersonate the role "Db" in order to generate database certificates, for more info see https://goteleport.com/docs/database-access/reference/cli/#tctl-auth-sign.`)
```